app/views/confirmables/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
Slim jau netika ielikts? Ja tika - tad varētu mēģināt visimaz vienu view nokonvertēt manuāli uz slim :)